integration-test/helpers/pages.js

```diff
@@ -309,13 +333,39 @@ export function loginPage (page, server, opts = {}) {
          */
         async assertClickAndFocusMessages () {
             const calls = await mockedCalls(page, ['showAutofillParent'])
-            expect(calls.length).toBe(2)
+            expect(calls.length).toBe(1)
```

Comment:
Can you explain how this is no-longer '2' - as mentioned in the comment I specifically added this test to ensure the functionality was captured.

If it no longer fire twice, great - but I'm not sure if macOS needs both?

Probably can be verified by testing scenarios (especially iFrames) on macOS

Reply:
It's because now it triggers automatically on page load because of autoprompt, so there's only one call.

Reply:
In fact, I should probably either remove that or change its name. Let me take another look.